src/vm/node_modules/proptable.js

```diff
@@ -1999,6 +1999,7 @@ exports.properties = {
         loadValueTranslator: 'utils.unbase64',
         payload: {
             allowed: {
+                'bhyve': ['create', 'receive', 'update'],
```

Comment:
The vmadm man page should also be updated to indicate that `vnc_password` is supported for both HVM types.

Reply:
Also copyright in this file?

Reply:
@bahamat yeah I noticed it too, see https://github.com/joyent/smartos-live/pull/900#issuecomment-582820863 as there are some additional things to mention/update. 

Anything I've missed?

Reply:
Ah, yeah. I missed that whole conversation. I was trying to hurriedly look over the change just before another meeting and didn't look at the comments. I think that text for the man page change looks good.